public/app/features/plugins/admin/components/PluginDetailsHeader.tsx

```diff
@@ -135,13 +135,23 @@ export const getStyles = (theme: GrafanaTheme2) => {
         &::after {
           content: '|';
           padding: 0 ${theme.spacing()};
+          color: ${theme.colors.text.primary};
         }
         &:last-child::after {
           content: '';
           padding-right: 0;
         }
       }
       font-size: ${theme.typography.h4.fontSize};
+
+      a {
+        color: ${theme.colors.text.link};
```

Comment:
Hmm... looking at the screenshots in the description I think this would look neater without the link color definitions and just the underline. WDYT?

Reply:
That make more sense. Thanks

Reply:
Changed so it only shows underline on hover, no color changes. Updated screenshots in the description

Reply:
Nice. I say we go with this. 👍